src/headers.ts

```diff
@@ -151,14 +157,23 @@ export function getCookies(route?: RouteContext): Promise<CookieAccessor> {
     });
 }
 
+/**
+ * @internal
+ */
 function isNextRequestHeaders(headers: PartialRequest['headers']): headers is NextRequest['headers'] {
     return headers.append !== undefined;
 }
```

Comment:
Some functions are marked as internal even though they aren't being exported

Reply:
It's right. It's internal for the lib consumers